### ISSUE TITLE
Feature Request: do not transform build-in require function

### ISSUE THREAD

**Issue by axetroy** [zeit/ncc#248]
Since `nuxt.config.js` is not packaged, the require will throw an error

```diff
const ROOT_DIR = rootDir || process.env.ROOT_DIR || process.cwd()
- const config = require(path.join(ROOT_DIR, 'nuxt.config.js'))
+ const config = __webpack_require__(7083)(path.join(ROOT_DIR, 'nuxt.config.js'))
```

Can we keep the built-in require?

For example by adding such a comment

```js
const config = /*ncc-build-in-require*/ require(path.join(ROOT_DIR, 'nuxt.config.js'));
```

Although this job seems to be webpack

But if do it with a plugin, that will be great.

**Comment by guybedford** [comment]
Certainly we do this already for other cases and this is an easy extension to existing logic we have for "non-analyzable requires".

**Comment by styfle** [comment]
Fixed in #378
